spec/integration/dry/rails/container/auto_register/with_block_spec.rb

```diff
@@ -5,8 +5,12 @@
 
   before(:all) do
     Dry::Rails.container do
-      auto_register!("app/workers") do |config|
-        config.memoize = true
+      begin
+        config.component_dirs.add("app/workers") do |dir|
+          dir.memoize = true
+        end
+      rescue Dry::System::ComponentDirAlreadyAddedError
+        # pass
```

Comment:
seems like `auto_register!` was a bit more forgiving about duplicates?

I think in specs this is fine, because in real-life nobody redefines container over and over (and in different order 😉)